File: 796. Rotate String/_796.py
class Solution:
    def rotateString(self, A: str, B: str) -> bool:
        if len(B)==len(A) and B in 2*A:
            return True
        return False

    def rotateString1(self, A, B):        
        return len(A) == len(B) and B in A + A

File: 796. Rotate String/796.go
package leetcode

import(
	"strings"
)

func rotateString(A string, B string) bool {
	tmp := strings.Join([]string{A, A}, "")
	if len(A) == len(B) && strings.Contains(tmp, B){
		return true
	} 
	return false
}

